### DISCUSSION TITLE
ESPAltherma suddenly stopped working

### DISCUSSION THREAD

**Discussion by andreapier** [raomin/ESPAltherma#208] [Q&A] [answered]
Hi,
my ESPAltherma suddenly stopped working 2 days ago.  
I don't know what happened, it was working just fine until then, but now it just show `Timeout error, check connections`.  
My wiring is fine because it worked for 1+ year, nonetheless I tried other wires but nothing changed.  
I also had a quick check of X10A port with a multimeter: both RX and TX pins are always showing 5V (with reference to ground pin), no fluctuations.
Could it be that my X10A is broken? Can someone please check if their X10A port is behaving the same?
Any other idea?

**Comment by raomin** [comment, marked as answer]
I guess you already tried to turn it off and on again...
I'll check the rx-tx voltage. But if there is nothing happening on the serial port, it makes sense that it is 5v steady.
Did you do something particular? A thunderstorm? 
Maybe your esp32 gpios are fried...?
Also, is your heatpump behaving correctly otherwise?

**Reply by raomin** [reply]
Yes it should be VCC level when idle:

> A signal at the VCC level (3.3V, 5V, etc.) indicates either an idle line, a bit of value 1, or a stop bit. A 0V (GND) signal represents either a start bit or a data bit of value 0.

**Reply by andreapier** [reply]
Thank for your time. Yes it is behaving just fine, and nothing notable happened.
I guess my ESP32 is just gone. I will try with another board.
Again, thanks!

**Comment by andreapier** [comment]
I setup another ESP32 an it is working just fine. I guess the old one just broke somehow.
